FabricExample/android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
+</adaptive-icon>
```

Comment:
Is there any particular reason we don't have trailing new lines at the end of the file? I checked and it's the same in Example - just wondering 🤔

Reply:
Those files are autogenerated by Android Studio when an image asset is created. The same thing happens when you create new resource file, i.e. there is no trailing new line.